src/main/java/seedu/duke/ui/Colors.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke.ui;
+
+/**
+ * Color class contains ANSI color codes for text color in the console.
+ */
+public class Colors {
```

Comment:
Maybe we could add a method to handle colored text. Example:  public static String colorize(String text, String colorCode) {
        return colorCode + text + ANSI_RESET;
    }